bbc/simorgh
# Live blog posting SEO updates

Part of https://jira.dev.bbc.co.uk/browse/WSTEAM1-680

Overall changes
======
- Modifies the schema structure of the `liveBlogPosting` object so that both `NewsArticle` and `LiveBlogPosting` are distinct objects

Testing
======
1. Visit http://localhost:7081/burmese/live/cx8jgp1zxylt?renderer_env=live
2. Open the page source by right clicking and clicking 'View Page Source'
3. Copy the entire page source
4. Go to https://validator.schema.org/
5. Click the 'Code Snippet' tab
6. Paste in the copied page source
7. Click 'Run Test'
8. Confirm that there are 2 entries, 'LiveBlogPosting' and 'NewsArticle'
9. Confirm there are no reported errors for either entry

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
